project-1/main.py

```diff
@@ -4,21 +4,61 @@
 from selenium.webdriver.common.by import By
 from selenium.webdriver.chrome.service import Service as ChromeService
 from webdriver_manager.chrome import ChromeDriverManager
+import os, json
+from dotenv import load_dotenv, find_dotenv
+load_dotenv()
+KEY = json.loads(os.getenv("PROJECT_1202"))
+
 
 driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()))
 link = "https://www.food.com/ideas/top-comfort-food-recipes-6929#c-791290"
 driver.get(url=link)
 
-gc = gspread.service_account(filename='my-project-1202.json')
+gc = gspread.service_account(KEY)
+
+
+def open_spreadsheet(sheet_name):
+    """
+    This opens a spreadsheet already created on Google sheet
+    """
+    sh = gc.open(sheet_name)
+    logging.info("spreadsheet opened")
+
+    def select_spreadsheet(s_name):
```

Comment:
You don't need to nest this function

Reply:
i used the nested function to open the current worksheet i am working with to get access to the worksheet variable.